.github/workflows/integration.yml

```diff
@@ -86,7 +90,14 @@ on:
           - letsencrypt_staging
           - letsencrypt_production
         default: letsencrypt_production
-
+      is_pre_installed:
+        type: choice
+        description: Fake if Otomi is pre-installed by Installer
```

Comment:
Why `Fake`? Do you mean `False`?

Reply:
This flag is supposed to be set by the APL-Installer, I added this option as a way to fake it for easier testing.

Reply:
Oh I read the sentence completely wrong, apologies.